modules/auxiliary/gather/ldap_hashdump.rb

```diff
@@ -86,7 +86,7 @@ def run_host(ip)
     entries_returned = 0
 
     print_status("#{peer} Connecting...")
-    ldap_new do |ldap|
```

Comment:
It looks like this now skips the logic that was explicitly put in to catch some bind errors:

https://github.com/rapid7/metasploit-framework/blob/dfbd14ea5b1de2d052a8580131c344863c855edb/lib/msf/core/exploit/remote/ldap.rb#L223-L255

Is that intentional?

Reply:
oh, I didn't spot that, not intentional no, I'll take another pass at this, would even just reverting this change and  removing the `if ldap.get_operation_result.code == 0` section be viable?